documentation/src/main/asciidoc/topics/con_file_based_cache_stores.adoc

```diff
@@ -4,13 +4,30 @@
 File-based cache stores provide persistent storage on the local host filesystem where {brandname} is running.
 For clustered caches, file-based cache stores are unique to each {brandname} node.
 
+For a file system that supports the Portable Operating System Interface (POSIX) standard, you can mount the file system to a pod on {openshiftshort} by using the same commands that you would use with the command line interface (CLI). 
+With this functionality, you can access storage capabilities that FCP, iSCSI, NFS, or Samba does not expose. 
+
+{brandname} defaults to using the `SoftIndexFileStore` implementation for file-based cache stores, because this implementation uses B+ trees that can improve how GC removes indexes before they consume too much memory.
+
+You can also consider the using the `SingleFileStore` implementation that persists data to file. This implementation might require additional memory for keeping an in-memory index of keys and the location of their values. 
+
+Other forms of cache stores exist, such as a filesystem-based cache store. 
```

Comment:
There are other cache-stores, but no filesystem-based ones.
Neither with SoftIndex nor SingleFileStore's should be used with NFS/Samba as they don't have POSIX standard.
So possible issues with the persistence are expected.

Reply:
Hi @wfink . 

The `filesystem-based cache stores` is incorrect? Should I mention a different name?

Reply:
singlefile and softindex are file based.
For me the sentence point back to it, with other stores I would point to JDBC/SQL or RocksDB

Reply:
Thanks, @wfink . Updated.